src/videojs.ima.js

```diff
@@ -1092,7 +1092,25 @@
       clearInterval(updateTimeIntervalHandle);
       clearInterval(seekCheckIntervalHandle);
       clearInterval(resizeCheckIntervalHandle);
-      player.one('play', player.ima.setUpPlayerIntervals_);
+      if(player.el()) player.one('play', player.ima.setUpPlayerIntervals_);
```

Comment:
Sorry for the additional comment - can we properly bracket this? It's against our style guidelines to have inline conditionals and statements:

if (conditional) {
  statement (even if it's just one line)
}